src/main/java/com/github/onsdigital/babbage/content/client/ContentClient.java

```diff
@@ -169,12 +173,31 @@ private ContentResponse resolveMaxAge(String uri, ContentResponse response) {
 
             if (timeToExpire == null) {
                 response.setMaxAge(maxAge);
+                //increment count of requests where publish date is not present
+                metrics.incPublishDateNotPresent();
             } else if (timeToExpire > 0) {
-                response.setMaxAge(timeToExpire < maxAge ? timeToExpire : maxAge);
+                // requested uri cache expiry is set as either:-
+                // 1. the time remaining until the publishing time or
+                // 2. the maximum cache expiry time permitted
+                if (timeToExpire < maxAge) {
+                    //increment count of requests where a publish date is present
+                    //and is not too far in the past or the future
+                    metrics.incPublishDatePresent();
+                    response.setMaxAge(timeToExpire);
+                } else {
+                    //increment count of requests where the timeToExpire is greater than or equal to the maximum/default value
+                    //permitted for the cache expiry time (maxAge) i.e. the publish date is too far in the future
+                    metrics.incPublishDateTooFarInFuture();
+                    response.setMaxAge(maxAge);
+                }
+                //increment count of requests where publish date is in the future
+                metrics.incPublishDateInFuture();
```

Comment:
should this metric and the one on line 185 be switched ?

Reply:
I think this one is correct where it is but the one on line 185 could be named publishDateNear instead of publishDatePresent perhaps?

Reply:
I've now removed this one and renamed the one on line 185 to be publishDateInRange if that's ok?